packages/manager/apps/ips/src/pages/listing/ipListing/ip.listing.page.tsx

```diff
@@ -72,6 +72,13 @@ export default function IpListingPage() {
             ]}
           />
           <QuickFilter />
+          <OdsButton
+            variant={ODS_BUTTON_VARIANT.outline}
+            icon={ODS_ICON_NAME.download}
+            size={ODS_BUTTON_SIZE.sm}
+            onClick={() => navigate(urls.listingExportIpToCsv)}
+            label=""
+          />
```

Comment:
I think we need to fix the design to have the same as the angular app
The button should be next to the settings button, on the same line so I think you should wrap the ActionMenu above and this OdsButton in a div with a className `flex gap-2`

![10_06_37](https://github.com/user-attachments/assets/c1d069e1-f04a-4595-bb15-8f29ef592e91)

It should look like this